src/components/view/DetailsTab.vue

```diff
@@ -33,6 +33,9 @@
             <router-link :to="{ path: '/volume/' + volume.uuid }">{{ volume.type }} - {{ volume.path }}</router-link> ({{ parseFloat(volume.size / (1024.0 * 1024.0 * 1024.0)).toFixed(1) }} GB)
           </div>
         </div>
+        <div v-else-if="$route.path.startsWith('/project/') && item === 'account' && fetchProjectAdmins()">
+          {{ fetchProjectAdmins() }}
```

Comment:
This line could cause (needs checking I'm not sure) the API to be called twice

Reply:
As part of fetchProjectAdmins I'm not calling the listProjectAccounts API again, but rather just filters the resource parameters

Reply:
Alright, let's merge this then.